index.html

```diff
@@ -2,13 +2,113 @@
+              <p class="presentation">Join well-known developers from around the world to explore how real-world Jamstack applications are built. Bring cutting-edge techniques back to your team.</p>
+              <div class="date">
+                <h2> 2021.10.22(Fri) ~ 24(Sun)</h2>
+                <p>@ Tunis Science City Boulevard Mohamed Bouazizi 1082 - BP. 114 Tunis 1004.</p>
+              </div>
+            </div>
 
+          </section>
+        </section> 
+        <section class="Program">
+           <h2 class="main-program"> Main Program</h2>
+           <hr>
+           <div class="events-wrapper">
+               <div class="event">
+                   <img src="statics/images/lecture_icone.png" alt="lecture">
+                   <h3 class="event-title"> Actionnable ideas</h3>
+                   <p class="event-description">
+                       Invest focused time levelling up on the emerging trends that should be part of your roadmap.
+                    </p>
+               </div> 
+               <div class="event">
+                   <img src="statics/images/exhibition.png" alt="exhibition">
+                   <h3 class="event-title"> EARLY ADOPTERS</h3>
+                   <p class="event-description">
+                      Discover new technical insights from software leaders pushing the boundaries.
+                    </p>
+               </div> 
+               <div class="event">
+                  <img src="statics/images/forum.png" alt="Forum">
+                  <h3 class="event-title"> Forum</h3>
+                  <p class="event-description">
+                    Hear from the Jamstack experts from Netlify, Microsoft, Stackbit, Bejamas, RedwoodJS, and others. Take part in live Q&As and Slack groups.
+                  </p>
+               </div> 
+               <div class="event">
+                 <img src="statics/images/workshop_icone.png" alt="Workshop">
+                 <h3 class="event-title"> Workshop</h3>
+                 <p class="event-description">
+                    Join well-known developers from around the world to explore how real-world Jamstack applications are built. Bring cutting-edge techniques back to your team.
+                 </p>
+               </div>
+               <div class="event">
+                <img src="statics/images/ignite.png" alt="Networking">
+                <h3 class="event-title"> Community</h3>
+                <p class="event-description">
+                    Make new friends and connect with people you never knew you missed! Relax, share, mix and mingle with like-minded folks throughout the Jamstack community.
+                </p>
+              </div>   
+           </div>
+           <p class="entire-program">SEE THE WHOLE PROGRAM</p>
+           <a class="btn-subscription" href="#">Join Jamstack Conference 2021</a>
+
+        </section>
+        <section class="speakers">
+
+        </section>
+        <footer >
```

Comment:
[ OPTIONAL ] Footer logo should be on the left and text on the right side instead of both in the center.